e2e/settings/index.spec.ts

```diff
@@ -64,7 +64,7 @@ test("it is possible to create and delete secrets", async ({ page }) => {
   await page.goto(`/${namespace}/settings`);
   await page.getByTestId("secret-create").click();
   const newSecret = {
-    name: faker.random.alphaNumeric(7),
+    name: faker.internet.domainWord(),
```

Comment:
@porleaaron I had to change this because the name needs to start with a letter. When the first letter is a number, it will fail to create the secret in the backend.

It also looks a bit nicer, in case we want to share screenshots or screencasts from our tests with other people.